webpack/dev.config.js

```diff
@@ -1,10 +1,9 @@
-require('babel/polyfill');
+require('babel-polyfill');
 
 // Webpack config for development
 var fs = require('fs');
 var path = require('path');
 var webpack = require('webpack');
-var WebpackIsomorphicTools = require('webpack-isomorphic-tools');
```

Comment:
But as it is not really related, I can squash and move this change as well as the (alphabetical) reordering of npm packages into a separate cleanup commit, should this be approved.